types/module/server/manager.go

```diff
@@ -149,7 +149,7 @@ func (c *configurator) ModuleKey() RootModuleKey {
 	return c.key
 }
 
-func (c *configurator) BinaryMarshaler() codec.BinaryMarshaler {
+func (c *configurator) Marshaler() codec.Marshaler {
```

Comment:
Do we need JSON now?

Reply:
yes for the ORM genesis import/export functions

Reply:
I see